testdata/baselines/reference/submodule/compiler/reachabilityChecks1.errors.txt

```diff
@@ -1,58 +1,77 @@
 reachabilityChecks1.ts(2,1): error TS7027: Unreachable code detected.
-reachabilityChecks1.ts(6,5): error TS7027: Unreachable code detected.
-reachabilityChecks1.ts(18,5): error TS7027: Unreachable code detected.
-reachabilityChecks1.ts(30,5): error TS7027: Unreachable code detected.
 reachabilityChecks1.ts(47,5): error TS7027: Unreachable code detected.
+reachabilityChecks1.ts(51,1): error TS7027: Unreachable code detected.
 reachabilityChecks1.ts(60,5): error TS7027: Unreachable code detected.
 reachabilityChecks1.ts(69,5): error TS7027: Unreachable code detected.
 
 
-==== reachabilityChecks1.ts (7 errors) ====
+==== reachabilityChecks1.ts (5 errors) ====
     while (true);
     var x = 1;
     ~~~~~~~~~~
-!!! error TS7027: Unreachable code detected.

```

Comment:
So are the changes here because the new logic does a better job of merging adjacent unreachable code blocks? Looks like a nice improvement.